login-workflow/src/screens/Login.tsx

```diff
@@ -128,7 +144,8 @@ export const Login: React.FC = () => {
     const authUIState = useAccountUIState();
     const { routes } = useRoutes();
     const theme = useTheme();
-    const classes = useStyles();
+    const { loginErrorDisplayConfig = { mode: 'dialog' } } = useInjectedUIContext();
```

Comment:
We are already calling this hook on line 165...you can destructure there so we don't call the hook twice.

Reply:
I need to destructure `loginErrorDisplayConfig` before defining classes and `loginHeader` after defining classes since it uses classes. So unless I'm missing some special syntax I think I'll have to make 2 calls regardless.

Reply:
You can call the hook once and then destructure in two stages:
```
const { loginError, ...otherUIContext } = useInjectedUIContext();
...
const { showRemmberMe, etc, etc, loginActions } = otherUIContext;
```